frontend/src/integrations/backend.tsx

```diff
@@ -54,15 +57,11 @@ export class Backend {
 export const BackendContext = React.createContext<Backend>(undefined!);
 
 export const BackendProvider: React.FC = ({ children }) => {
-  const { access } = useAuth();
-  const [config, setConfig] = useState<AxiosRequestConfig>({});
-
-  useEffect(() => {
-    if (access) {
-      setConfig({ headers: { Authorization: `Bearer ${access}` } });
-    }
-  }, [access]);
-
+  const { access, refresh } = useAuth();
+  const config : BackendConfig = {
+    headers: { Authorization: `Bearer ${access}` },
+    refresh,
+  };
```

Comment:
No estoy seguro si esto se va a ejecutar con cada renderizado, y es una operación relativamente costosa en memoria, si no queremos usar un estado, podemos usar `useMemo` para no construir `config` en cada render:
```suggestion
  const config : BackendConfig = useMemo(() => ({
    headers: { Authorization: `Bearer ${access}` },
    refresh,
  }), [access, refresh]);
```